lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -71,6 +71,8 @@ class SPIRVModuleImpl : public SPIRVModule {
     MemoryModel(MemoryModelOpenCL){

```

Comment:
If remove this the function addCapability() will not be called and disappears "capability kernel" in output disassembled "spirv_file"